### ISSUE TITLE
Circular dependency in testclasses.php

### ISSUE THREAD

**Issue by mindplay-dk** [TomBZombie/php-dependency-injection-benchmarks#18]
It looks like you have a circular dependency [here](https://github.com/TomBZombie/php-dependency-injection-benchmarks/blob/master/testclasses.php#L88), e.g. `K` depends on `K`.

Is that deliberate?

**Comment by njasm** [comment]
@mindplay-dk I've also notice that when I was optimizing the tests #17  for the latest version of my DiC .
but i didn't find any test actually using the K class... 

maybe @TomBZombie  could share some light on the matter?

**Comment by mindplay-dk** [comment]
Yeah, looks like it's actually unused, I noticed that as well - probably just left there by accident.

**Comment by TomBZombie** [comment]
Indeed, I did have a benchmark which looked at containers handling circular dependencies, however as not all of them actually support it I had to remove it entirely. I'll remove the unused class.
